src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const male = sex === 'm';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {(Boolean(age)) && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      {(!isMarried) && (
+        <p className="Person__partner">I am not married</p>
+      )}
+      {
+        <p className="Person__partner">
+          {(male)
+            ? `${partnerName} is my wife`
+            : `${partnerName} is my husband`
+          }
+        </p>
+      }
```

Comment:
Use ternary operator to avoid such issue

![Screenshot 2023-04-06 at 10 57 41](https://user-images.githubusercontent.com/54940595/230312686-a87f090c-b517-4599-aaec-0b1d7d7ca1a8.png)